### PR TITLE
Add some more debugging information

### PR DESCRIPTION
**Pull request by jez**
### Motivation

These functions are handy to have in a debugger.

### Test plan

See included automated tests.

### CODE REVIEW

**Review by elliottt — APPROVED** [review]

File: core/Files.h [hunk 1]
```diff
@@ -19,6 +19,7 @@ class FileRef final {
     FileRef() : _id(0){};
     FileRef(unsigned int id);
 
+    FileRef(FileRef &f) = default;
```
Comment (froydnj) [inline core/Files.h:22]: Why is this needed?  Does something in the debugger not do proper resolution to select the `const &` overload?

Comment (jez) [reply core/Files.h:22]: Yeah, I kept getting warnings in lldb until I added this.
